dev-requirements.txt

```diff
@@ -32,10 +32,13 @@ more-itertools>=4.0.0; python_version >= '3.6'
 # safety 1.9.0 removed support for Python 2.7 (and now also enforces that)
 safety>=1.8.7,<1.9.0; python_version == '2.7'
 safety>=1.9.0; python_version >= '3.5'
+safety>=2.2.0; python_version >= '3.6'
```

Comment:
If you add this for py>=3.6, you need also to limit the previous line to py<3.6, i.e. py==3.5.

Reply:
DONE